### PR TITLE
[Shop] Reorganize Twig hooks on login page

### PR DESCRIPTION
**Pull request by loic425**
| Q               | A
|-----------------|-----
| Branch?         | 2.0
| Bug fix?        | no
| New feature?    | no
| BC breaks?      | no
| Deprecations?   | no
| Related tickets | fixes 
| License         | MIT

Same issue as #17344, the form is started too soon. I've started the form on login container.

image

### CODE REVIEW

File: src/Sylius/Bundle/ShopBundle/Resources/config/app/twig_hooks/account/login.yaml [hunk 1]
```diff
@@ -1,47 +1,52 @@
 sylius_twig_hooks:
     hooks:
         'sylius_shop.account.login':
-            form:
-                template: '@SyliusShop/account/login/form.html.twig'
-                priority: 0
+            content:
+                template: '@SyliusShop/account/login/content.html.twig'
+                priority: 0
```
Comment (diimpp) [inline src/Sylius/Bundle/ShopBundle/Resources/config/app/twig_hooks/account/login.yaml]:
```suggestion
                priority: 0
```

File: src/Sylius/Bundle/ShopBundle/Resources/config/app/twig_hooks/account/login.yaml [hunk 2]
```diff
@@ -1,47 +1,52 @@
...
-                template: '@SyliusShop/account/login/form/register_container.html.twig'
+                template: '@SyliusShop/account/login/content/register_container.html.twig'
                 priority: 0
 
-        'sylius_shop.account.login.form.login_container':
+        'sylius_shop.account.login.content.login_container':
             header:
-                template: "@SyliusShop/account/login/form/login_container/header.html.twig"
-                priority: 400
+                template: "@SyliusShop/account/login/content/login_container/header.html.twig"
```
Comment (diimpp) [inline src/Sylius/Bundle/ShopBundle/Resources/config/app/twig_hooks/account/login.yaml]: Not sure which syntax `'` or `"` is official, but file uses both.

```suggestion
                template: '@SyliusShop/account/login/content/login_container/header.html.twig'
```

File: src/Sylius/Bundle/ShopBundle/templates/account/login/form.html.twig
Comment (diimpp) [inline src/Sylius/Bundle/ShopBundle/templates/account/login/form.html.twig:1]: Should this file be removed? I think it's empty now.

Comment (loic425) [reply src/Sylius/Bundle/ShopBundle/templates/account/login/form.html.twig:1]: The file is removed, it's on account/login/content/login_container

**Review by diimpp — APPROVED** [review]

**Review by GSadee — APPROVED** [review]

**Pull request merged**